src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewCell.swift

```diff
@@ -36,10 +36,31 @@ struct DynamicCell {
 extension DynamicCell {
 	enum CellReuseIdentifier: String, TableViewCellReuseIdentifiers {
 		case dynamicTypeText = "regularCell"
+		case dynamicTypeTextView = "textViewCell"
 		case icon = "iconCell"
 		case space = "spaceCell"
 	}
 
+	/// Style of  `DynamicTableViewTextCell`
+	///
+	/// These now come in two flavors:
+	/// - `UILabel` backed
+	/// - `UITextView` backed
+	enum TextCellStyle {
+		/// DynamicCell with a basic label
+		case label
+		/// DynamicCell that uses a UITextView with data recognizers instead of a UILabel to display text
+		/// Useful for automatic link, phone #, etc. recognization
+		case textView(dataDetectors: UIDataDetectorTypes)
```

Comment:
Personally I would remove the label for the associated value to be less verbose in code.